src/appleseedmaya/CMakeLists.txt

```diff
@@ -159,8 +155,14 @@ target_link_libraries (appleseedMaya
     ${PYTHON_LIBRARIES}
 )
 
-set_target_properties (appleseedMaya PROPERTIES PREFIX "")
+set_target_properties (${MAYASEED_PLUGIN_NAME} PROPERTIES
```

Comment:
I prefer if we don't do this. It's important when scripting that the plugin has a fixed name.